sdk/tables/Azure.Data.Tables/src/Generated/Models/TableGeoReplicationInfo.Serialization.cs

```diff
@@ -11,9 +11,9 @@
 
 namespace Azure.Data.Tables.Models
 {
-    public partial class TableGeoReplication
+    public partial class TableGeoReplicationInfo
```

Comment:
Probably out of the scope of this PR, but the `Info` suffix here seems like it would not pass architecture review.

Reply:
I'm going to punt on this for now.

Reply:
It matches a type of the same name and purpose in Storage so I think it'd be fine like this.  We used to use `Info` for a lot of things, but have collectively switched to `Details` per https://azure.github.io/azure-sdk/general_design.html#model-types.